### ISSUE TITLE
Remove `position: fixed`

### ISSUE THREAD

**Issue by vjeux** [excalidraw/excalidraw#2737]
In the embeddable version of excalidraw, we should not use position: fixed as it assumes it takes the entire viewport which is not a correct assumption.

I don't remember why I used position fixed in the first place, so there may be dragons, but I don't see any hard limitations where that couldn't work out without. The biggest one I see is the wysiwyg text but we should be able to put `overflow: hidden` on the container to avoid the scrollbar issue.

**Comment by vjeux** [comment]
@dwelle told me about a workaround. If you put `transform: scale(1)` on a scrollable container, it will prevent it to be fixed against the entire page.

**Comment by vjeux** [comment]
The entire hack:
```js
    <div
      style={{
        border: '1px black solid',
        width: `${width}px`,
        height: `${height}px`,
        transform: 'scale(1)',
      }}>
      <Excalidraw
        width={width}
        height={height}
        offsetLeft={0}
        offsetTop={0}
        user={{name: 'Excalidraw User'}}
      />
    </div>
```

**Comment by vjeux** [comment]
Actually it doesn't work as we don't properly compute the offset for mouse interactions.

**Comment by ad1992** [comment]
closed via #3018
